java/apps/authentication-portal/src/main/webapp/includes/branding-preferences.jsp

```diff
@@ -69,7 +69,7 @@
 
         // If the `sp` param is defined, set the resource type as app.
         if (StringUtils.isNotBlank(applicationRequestingPreferences)) {
-            String appWiseBrandingPreferenceFilePath = getServletContext().getRealPath("extensions/branding/"+ APP_PREFERENCE_RESOURCE_TYPE +"/" + applicationRequestingPreferences + "/en-US/branding-preference.json");
+            String appWiseBrandingPreferenceFilePath = getServletContext().getRealPath("extensions/branding/" + tenantRequestingPreferences + "/" + "apps" + "/" + applicationRequestingPreferences + "/branding-preference_en_US.json");
```

Comment:
Let's parameterize this part as well.
In a followup PR lets try to grab the selected locale and check for the localization file.

https://github.com/wso2/identity-apps/blob/master/java/apps/recovery-portal/src/main/webapp/includes/language-switcher.jsp#L39

```suggestion

String locale = "en_US";

            String appWiseBrandingPreferenceFilePath = getServletContext().getRealPath("extensions/branding/" + tenantRequestingPreferences + "/" + "apps" + "/" + applicationRequestingPreferences + "/branding-preference_" + locale + ".json");
```

Reply:
Added a TODO comment for this